libs/libcommon/src/libcommon/config.py

```diff
@@ -29,21 +29,58 @@
 from libcommon.processing_graph import ProcessingGraphSpecification
 
 ASSETS_BASE_URL = "assets"
-ASSETS_STORE_DIRECTORY = None
+ASSETS_STORAGE_DIRECTORY = None
 
 
 @dataclass(frozen=True)
 class AssetsConfig:
     base_url: str = ASSETS_BASE_URL
-    storage_directory: Optional[str] = ASSETS_STORE_DIRECTORY
+    storage_directory: Optional[str] = ASSETS_STORAGE_DIRECTORY
```

Comment:
the changes in environment variables (renaming, addition) must be reflected in the docker compose files, and in the Helm chart

Reply:
Done in the docker compose and in `_envAssets.tpl`

Reply:
it seems like you didn't push your commit

Reply:
To be more clear: I only renamed the python variable. The env variable was already `ASSETS_STORAGE_DIRECTORY`

Reply:
OK!

Still: the new environment variables are not in the chart